src/System/Process/PID1.hs

```diff
@@ -202,6 +202,7 @@ runAsPID1 cmd args env' timeout = do
         case p_ of
             ClosedHandle e -> assert False (exitWith e)
             OpenHandle pid -> return pid
+            OpenExtHandle{} -> error "OpenExtHandle is a Windows concept, and pid1 is not designed to run on that"
```

Comment:
hey @nh2 I don't think you meant to add this to the PR, right?

Reply:
It is intended, see commit message https://github.com/denibertovic/pid1/pull/1/commits/402d085804da8299ada15e1eb5b33e0fba352a3f

Reply:
:+1: